Lib/fontParts/objects/base/validators.py

```diff
@@ -115,10 +115,11 @@ def validateLayerName(value):
 # -----
 
 def validateGlyphName(value):
-    """
-    XXX implement
-    """
-    return value
+    if not isinstance(value, basestring):
+        raise FontPartsError("Glyph names must be unicode strings, not %s." % type(value).__name__)
```

Comment:
basestring is either str or unicode type in py2 (unicode string in py3). Glyph names in most cases contain only ascii. I guess it's ok if they are instances of basestring, so the error message should warn that they "must be strings" (in a general sense) rather than "must be unicode strings". Unless we want to assert that they really are isinstance(value, unicode)?